packages/nimble-components/src/table-column/anchor/tests/table-column-anchor.stories.ts

```diff
@@ -0,0 +1,155 @@
+    parameters: {
+        docs: {
+            description: {
+                component: columnTypeOverviewText
+            }
+        }
+    },
+    // prettier-ignore
+    argTypes: {
+        ...sharedTableArgTypes,
+        selectionMode: {
+            table: {
+                disable: true
+            }
+        },
+    }
+};
+
+export default metadata;
+
+const simpleData = [
+    {
+        firstName: 'Ralph',
+        lastName: 'Wiggum',
+        url: 'https://www.google.com/search?q=ralph+wiggum'
+    },
+    {
+        firstName: 'Milhouse',
+        lastName: 'Van Houten',
+        url: 'https://www.google.com/search?q=milhouse+van+houten'
+    },
+    {
+        firstName: 'Ned',
+        lastName: 'Flanders',
+        url: 'https://www.google.com/search?q=ned+flanders'
+    },
+    {
+        firstName: 'Maggie (no link)',
+        lastName: 'Simpson'
+    },
+    {
+        lastName: 'Simpson',
+        url: 'https://www.google.com/search?q=simpsons'
+    },
+    {
+        lastName: 'Simpson'
+    }
+];
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface AnchorColumnTableArgs extends SharedTableArgs {
+    labelFieldName: string;
+    hrefFieldName: string;
+    placeholderText: string;
+    appearance: keyof typeof AnchorAppearance;
+}
+
+const anchorColumnDescription = `The \`nimble-table-column-anchor\` column is used to display string fields as links or text in the \`nimble-table\`. If a row provides an href for a link, that cell will display a link, otherwise it will display plain text.
+
+In an Angular app, \`nimble-table-column-anchor\` can integrate with the router by applying the attribute \`nimbleRouterLink\`. This will cause the router to be invoked when navigating to the paths given in the table data. Other properties supported by the \`RouterLink\` directive (e.g. \`queryParams\`, \`state\`, \`replaceUrl\`) can also be set on \`nimble-table-column-anchor\`.
+\`\`\`
+<nimble-table-column-anchor nimbleRouterLink [queryParams]="{ id: 'foo' }" replaceUrl>
+    Link
+</nimble-table-column-anchor>
+\`\`\`
+Note that these property values apply to every link in the column and cannot be configured on a per-link basis.
+`;
+
+export const anchorColumn: StoryObj<AnchorColumnTableArgs> = {
+    parameters: {
+        docs: {
+            description: {
+                story: anchorColumnDescription
+            }
+        }
+    },
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html<AnchorColumnTableArgs>`
+        ${usageWarning('table')}
+        <${tableTag}
+            ${ref('tableRef')}
+            data-unused="${x => x.updateData(x)}"
+        >
+            <${tableColumnAnchorTag}
+                label-field-name="${x => x.labelFieldName}"
+                href-field-name="${x => x.hrefFieldName}"
+                placeholder="${x => x.placeholderText}"
+                appearance="${x => x.appearance}"
+            >
+            First Name
```

Comment:
nit: It's a bit confusing to have this column always be titled "First Name" when it could display either the first name or last name based on the controls within the story. I think it would be better to name it something more generic like "link" or "link column" to be explicit that it's the part of the table the story controls.

Reply:
Changed to "Link Column"